.project

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is used by users on Eclipse. Why has it been deleted?

Reply:
It caused errors in the travis and appveyor runs because it's been added to the .gitignore file. the check:git_ignore now fails if you don't remove that file.

Reply:
Add the following to your .sync.yml and it will fix the issues you are seeing. We have encountered this problem with all the other modules too:
```
.gitignore:
  required:
    - ---.project
```

Reply:
Had to run a pdk update after the add, also.